### PR TITLE
Update manifest.json

### PR DESCRIPTION
**Pull request by narwhql**
Changed wildcard and added tiktok

## Type of change

- [ ] Add new directory for a server
- [X] Update directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values
- [ ] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc

### CODE REVIEW

File: minecraft_servers/nektax/manifest.json [hunk 1]
```diff
@@ -2,7 +2,11 @@
     "server_name": "nektax",
     "nice_name": "Nektax Network",
     "direct_ip": "nektax.net",
-    "server_wildcards": ["play.nektax.net"],
+    "server_wildcards": [
+        "play.nektax.net",
+        "premium.nektax.net",
```
Comment (JakobBruening) [inline minecraft_servers/nektax/manifest.json:7]: You can remove both, as "%.nektax.net" includes the wildcards.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/nektax/manifest.json [hunk 2]
```diff
@@ -11,7 +13,8 @@
     "social": {
         "web_shop": "https://nektax.tebex.io",
         "web_support": "https://discord.gg/zY7X4rVWEb",
```
Comment (DoJapHD) [inline minecraft_servers/nektax/manifest.json:15]: web_support can't be Discord, link to FAQ; if non-existent, remove

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by narwhql** [comment]
Removed, strange that I had added it a while ago and it was accepted...
Anyway, should be good now 👍🏻

**Comment by DoJapHD** [comment]
> Removed, strange that I had added it a while ago and it was accepted... Anyway, should be good now 👍🏻

Probably just overlooked

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**